### PR TITLE
[Tel-255] provide context to the first call participant

### PR DESCRIPTION
**Pull request by ivanjiang5628**
## Description
* Add waiting message to the `Watermark` component.
* Add `ProximaNovaLtLight` fonts to Jitsi `web` app.
* Remove the `reconnect` button.
* Use `HtmlWebpackPlugin` to generate the random number after js assets src URL for cache breaking
* This feature also requires Jane's `video_chat_sessions` JWT token context update:
https://github.com/janeapp/jane/pull/5262

[Jira Link](https://janeapp.atlassian.net/browse/TEL-255)

## Code Review
Resource: [Dev Team Notion Page](https://www.notion.so/janeapp/Dev-Team-f06c6eb2ccca4066bc63fc1ac1bd2549)
Resource: [Code Review Checklist](https://www.notion.so/janeapp/Code-Review-checklist-2c510c527ac7470c902a5e8f25f9db3c)

- [x] I clearly explained the WHY behind the work, in the Description above

#### Design
- [x] I added instructions for how to test, in the QA section below
- [ ] I added tests for changes, or determined that none were required
- [ ] I demoed this to the appropriate person
- [x] I considered both mobile & desktop views, or that wasn't relevant

#### Code
- [x] I committed code with informative git messages
- [x] I wrote readable code, or added comments if it was complex
- [x] I performed a self-review of my own code
- [x] I rebased my branch on the latest `master`
- [ ] I confirmed that all CircleCI tests are passing
- [x] I didn't add new npm packages, or else I made damn sure the `yarn.lock` changes are safe for existing production packages

## QA and Smoke Testing
create an online appointment -> join and wait for another participant -> the messages will be shown.

## Screenshots
- A participant joins the call after the appointment started and not seeing another participant.

https://www.loom.com/share/e223322ad12944588de40128db299a3b

- A participant joins the call ahead of the appointment time (the message will show the `appointment start time`)
- Wait until the appointment has started and not seeing another participant. The message will change.

https://www.loom.com/share/96cf696cb1ad475eba1c3bc738087090

- when the second participant joins and leaves the call 

https://www.loom.com/share/6d19262cf2924e418b8d51d8cb883957

- Mobile layout:
https://www.loom.com/share/6e62d190a8fd45ec80ca5cdde63f53f6

- Mobile flow (pre-start -> post-start time -> the second participant joins):
https://www.loom.com/share/4c08230818c14bcbb7e272d627b44afd

- Animation update: only Logo fades in and out:

https://www.loom.com/share/5aa26dbba02c422b8ae7cf68df2321f6

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,160 +1 @@
...
...ent.getElementById('reloadLink'); reloadLink.setAttribute('href', href); var showMoreElem=document.getElementById("showMore"); showMoreElem.addEventListener('click', function (){var moreInfoElem=document.getElementById("moreInfo"); if (showMoreElem.innerHTML===showMoreText){moreInfoElem.setAttribute( "style", "display: block;" + "color:#FF991F;" + "font-size:small;" + "user-select:text;"); showMoreElem.innerHTML=showLessText;}else{moreInfoElem.setAttribute( "style", "display: none;"); showMoreElem.innerHTML=showMoreText;}}); window.setTimeout( function (){window.location.replace(href);}, delay); // Call extra handler if defined. if (typeof postLoadErrorHandler==="function"){postLoadErrorHandler(fileRef);}}; window.removeEventListener( 'error', loadErrHandler, true /* capture phase */);}}; window.addEventListener( 'error', loadErrHandler, true /* capture phase type of listener */); </script> <script></script> <script src="libs/do_external_connect.min.js?v=2"></script> <script></script> <script></script> <script src="<%=htmlWebpackPlugin.options.jitsiLib %>"></script> <script src="<%=htmlWebpackPlugin.options.appBundle %>"></script> </head> <body> <div id="react"></div></body></html>
```
Comment (ivanjiang5628) [inline index.html:1]: Just minified this HTML file...

File: react/features/conference/components/web/Conference.js [hunk 2]
```diff
@@ -190,7 +190,7 @@ class Conference extends AbstractConference<Props, *> {
                 id = 'videoconference_page'
                 onMouseMove = { this._onShowToolbar }>
                 <Notice />
-                <RefreshButton />
+                {/*<RefreshButton />*/}
```
Comment (gerbus) [inline react/features/conference/components/web/Conference.js:193]: Leaving this for later? Might be worth a note about why it's left hanging around.

File: webpack.config.js [hunk 3]
```diff
@@ -22,7 +24,8 @@ const minimize
  */
 function getPerformanceHints(size) {
     return {
-        hints: minimize ? 'error' : false,
+        // hints: minimize ? 'error' : false,
+        hints: minimize ? 'warning' : false,
```
Comment (gerbus) [inline webpack.config.js:28]: Do you need the old one left as a comment?

File: react/features/base/react/components/web/WaitingMessage.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+    _startTimer() {
+        const {jwt, conferenceHasStarted} = this.props;
+        const jwtPayload = jwt && jwtDecode(jwt);
+        if (jwtPayload && jwtPayload.context && !conferenceHasStarted) {
+            const {start_at} = jwtPayload.context || 0;
+            const appointmentStartTimeStamp = new Date(start_at).getTime();
+            const now = new Date().getTime();
+
+            if (appointmentStartTimeStamp - now > 0) {
```
Comment (gerbus) [inline react/features/base/react/components/web/WaitingMessage.js:49]: `now < appointmentStartTimeStamp` might be easier to read?

File: react/features/base/react/components/web/WaitingMessage.js [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+                    this._setInterval(appointmentStartTimeStamp);
+                });
+            }
+        }
+    }
+
+    _setInterval(appointmentStartTimeStamp) {
+        this._interval = setInterval(() => {
+            const {conferenceHasStarted} = this.props;
+            if ((appointmentStartTimeStamp - new Date().getTime() < 0) || conferenceHasStarted) {
```
Comment (gerbus) [inline react/features/base/react/components/web/WaitingMessage.js:63]: Might be easier to read:
```
const now = new Date().getTime();
if ((appointmentStartTimeStamp < now) || conferenceHasStarted) {
```

File: react/features/base/react/components/web/WaitingMessage.js (hunk 4 above)
Comment (ivanjiang5628) [reply react/features/base/react/components/web/WaitingMessage.js:49]: Yes! You are right!

File: react/features/conference/components/web/Conference.js (hunk 2 above)
Comment (ivanjiang5628) [reply react/features/conference/components/web/Conference.js:193]: Hmm Yes.I think I should remove it, will change 😃

**Pull request merged**